dev-docs/modules/userId.md

```diff
@@ -594,6 +594,11 @@ Please reach out to [FabrickIntegrations@team.neustar](mailto:FabrickIntegration
 | params.m | | String | This is a mobile advertising ID (IDFA/AAID) used to link a user to their Fabrick ID. | |
 | params.ia | | String | This is an identifier for advertising (IFA) used to link a user to their Fabrick ID. | |
 | params.iv | | String | This is an identifier for vendors (IFV) used to link a user to their Fabrick ID. | |
+| params.1pd | | String | This is the 1st party user ID (e.g. a Customer ID/CUSTID) - Note: Requires separate delivery of identity log files keyed off the 1st party user ID to establish an identity sync. | |
+| params.u | | String | This is the page_url - the url which the user is currently browsing. * encoding required for any character outside of alphabets (A-Z a-z), digits (0-9), hyphen (-), underscore (_) tilde (~), and dot (.). | |
+| params.f | | String | This is the referrer_url - the url which the user visited prior to landing on the page_url. * encoding required for any character outside of alphabets (A-Z a-z), digits (0-9), hyphen (-), underscore (_) tilde (~), and dot (.). | |
+| params.ifa_type | | String | This denotes the source of the IFA. Please refer to https://iabtechlab.com/wp-content/uploads/2018/12/OTT-IFA-guidelines.final_Dec2018.pdf for recommended values and additional details. | |
+| params.lmt | | Boolean | Possible values are '0' or '1'. A value of '1' indicates that a user has requested that ad tracking and measurement be disabled. If a value of '1' is being passed, the real IFA must not be sent via the 'ia' parameter – a 'synthetic' or 'session' IFA can be sent. Please refer to https://iabtechlab.com/wp-content/uploads/2018/12/OTT-IFA-guidelines.final_Dec2018.pdf for recommended values and additional details for additional details. | |
```

Comment:
Please refer to the comment above regarding how to add a link. Also could you please remove the repeated "for additional details" at the end of the sentence.